introduction.md

```diff
@@ -1,3 +1,10 @@
 Hello my name is Frank and i think You know me from Donnie Darko.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?